vendor/ppbtlib/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
file to remove. add by the package script

Reply:
removed